web/modules/custom/dpl_patron_reg/src/DplPatronRegSettings.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\dpl_patron_reg;
+
+use Drupal\dpl_react\DplReactConfigBase;
+
+/**
+ * Class that handles patron registration settings.
+ */
+class DplPatronRegSettings extends DplReactConfigBase {
+
+  /**
+   * Gets the configuration key for the instant patron registration settings.
+   */
+  public function getConfigKey(): string {
+    return 'dpl_instant_loan.settings';
```

Comment:
This should be a config key reserved for dpl patron settings

Reply:
My bad

Reply:
Fixed